language-plutus-core/src/Language/PlutusCore/Lexer.x

```diff
@@ -93,6 +93,7 @@ tokens :-
     <0> equalsByteString         { mkBuiltin EqByteString }
     <0> txhash                   { mkBuiltin TxHash }
     <0> blocknum                 { mkBuiltin BlockNum }
+    <0> blocknum                 { mkBuiltin SizeOfInteger }
```

Comment:
Cut-copy-paste typo

Reply:
Err, that stupid error would be annoying to debug. Thank you.